### DISCUSSION TITLE
Error while running ldt.parameter file

### DISCUSSION THREAD

**Discussion by Arijit22IITDelhi** [NASA-LIS/LISF#1382] [Q&A] [answered]
Hello, I was running **ldt.parameter** file using **LDT** and PRINCETON met forcing for irrigation parameterization, it is showing this kind of error:

**Error:**
forrtl: No such file or directory
forrtl: severe (29): file not found, unit 15, file /home/civil/msr/cey227525/LIS-IND-SMAP/LIS_smap/fort.15
Image              PC                Routine            Line        Source
LDT                0000000000A4283B  Unknown               Unknown  Unknown
LDT                0000000000A7B002  Unknown               Unknown  Unknown
LDT                00000000009AFEC5  readcropinventory          78  readcropinventory.F90
LDT                000000000066427F  monfredaetal08_cr         166  Monfredaetal08_crops_module.F90
LDT                00000000004F8E51  ldt_lsmcropmodifi         305  LDT_LSMCropModifier_Mod.F90
LDT                00000000005FF6C6  ldt_paramprocmod_         294  LDT_paramProcMod.F90
LDT                00000000005A1E70  ldt_init_lsmparam          24  LDT_init_LSMparamproc.F90
LDT                0000000000655A60  MAIN__                     46  LDTmain.F90
LDT                0000000000416DB2  Unknown               Unknown  Unknown
libc-2.17.so       00002B9F3C82D555  __libc_start_main     Unknown  Unknown
LDT                0000000000416CA9  Unknown               Unknown  Unknown

It is showing that there is no such file named fort.15, in fact I haven't used any file with such name in the config file. 
Please help me in resolving the issue.
I am sharing the corresponding ldt config file and ldtlog file for reference.

[LDTlog.txt](https://github.com/NASA-LIS/LISF/files/12066485/LDTlog.txt)
[ldt.parameter.txt.txt](https://github.com/NASA-LIS/LISF/files/12066486/ldt.parameter.txt.txt)

**Comment by dmocko** [comment, marked as answer]
Hi @Arijit22IITDelhi 

First, can you try again, but remove the double-quotes before/after your directory paths?:

```
Crop library directory:  ./INPUT/LS_PARAMETERS/crop_params/Crop.Library.Files/
Crop type file:          ./INPUT/LS_PARAMETERS/crop_parms/Monfreda_175Crops/Crops/
```

From your error message, the code is stopping at Line 78 in this routine I am highlighting:

https://github.com/NASA-LIS/LISF/blob/master/ldt/params/crop_parms/readcropinventory.F90#L64-L78

It's likely the filename is not being set correctly (such as due to the double-quotes that should not be there).

Also, do you have this file on disk available from the directory in which you are running LDT?

`./INPUT/LS_PARAMETERS/crop_params/Crop.Library.Files/FAOSTAT05_Crop.Inventory`

**Reply by Arijit22IITDelhi** [reply]
Thank you so much. I worked successfully.
